scripts/out

```diff
@@ -8,6 +8,10 @@ import urllib
 from bitbucket import BitbucketException, BitbucketCloudDriver, BitbucketServerDriver, set_build_status
 from concourse import print_error
 from helper import json_pp
+try:
```

Comment:
hello @mockersf, for my understanding: why is there the need to select based on the Python version, since the script forces python3 with the shebang ?

Reply:
actually none, it's just the way I'm used to do it...